src/engineer/__init__.py

```diff
@@ -1,3 +1,4 @@
 from .engineer import Engineer
```

Comment:
No need to import the Engineer, since it is only used when its inherited by the other classes